django_cte/query.py

```diff
@@ -66,16 +74,35 @@ def chain(self, klass=None):
 class CTECompiler(object):
 
     @classmethod
-    def generate_sql(cls, connection, query, as_sql):
+    def generate_sql(cls, connection, query, with_col_aliases, as_sql):
         if query.combinator:
             return as_sql()
 
         ctes = []
         params = []
+        if django.VERSION > (4, 2):
```

Comment:
`>=` would be less misleading. While the actual `VERSION` tuple for 4.2 looks like `(4, 2, 0, 'final', 0)`, a human reader might assume this is intended to exclude 4.2.

(Same on line 162.)